examples/browser-sprite-extract-mode/webpack.config.js

```diff
@@ -19,7 +19,7 @@ module.exports = merge(baseConfig, {
         test: /\.svg$/,
```

Comment:
Let's just remove this examples folder - it doesn't make sense to us

Reply:
Removed. But this did not reduce the number of issues. There are no packages specific for this example.

Reply:
Why do we need `react` dependency now, after removing an `examples` folder? Is it used anywhere else?
That's why I asked "Will it reduce number of dev deps".

Could you carefully go through all the dependencies and remove the unused ones please.

Reply:
I hope I removed all unused code and packages.

Reply:
Great, thanks 👍